articles/clean-and-cleaner-code.md

```diff
@@ -0,0 +1,27 @@
+---
+authorName: Milan Milanovic
+authorGithubUsername: milanm
+issue: 59
+title: Clean and cleaner code
+---
+# {{page.title}}
+
+Being a developer in these days means to develop features and fix bugs for a customer sometimes in a fast pace, which may result in a badly maintainable code. On the other hand, we also want to make the most elegant solutions to some problems. This can produce code that machine can understand but people cannot so well. Such code can become a mayor issue for a company to maintain and use in a long run. 
```

Comment:
+ Not only for the "customer" - just remove that 😊

Reply:
Fixed.